raft/server.cc

```diff
@@ -214,6 +216,14 @@ class server_impl : public rpc_server, public server {
     };
     absl::flat_hash_map<server_id, append_request_queue> _append_request_status;
 
+    struct api_requests {
```

Comment:
Why be so specific as to call them `api`? What if we will want to process internal server requests this way as well?